test/exercism/metrics_test.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../test_helper'
+require 'active_support'
+require 'active_support/core_ext/object'
+require 'exercism/metrics'
+require 'mocha/setup'
+
+class MetricsTest < MiniTest::Test
+  def test_reports_without_error
+    metrics = Metrics.new(api_key: 'test', host: 'localhost')
+    UDPSocket.any_instance.expects(:send)
```

Comment:
I think this is provided by `mocha`, it has a bunch of interesting assertions. :)